<launch>
    <node name="multi_filter" pkg="filters" type="multi_filter" output="screen">
        <param name="subscribed_topic" type="string" value="/point_cloud/cloud_registered"/>
        <param name="published_topic" type="string" value="cloud_filtered"/>
        # Voxel Grid Filter parameters
        <param name="leaf_size_x" type="double" value="0.05"/>
        <param name="leaf_size_y" type="double" value="0.05"/>
        <param name="leaf_size_z" type="double" value="0.15"/>
        # PassThrough Filter parameters
        <param name="field_name" type="string" value="x"/>
        <param name="min_value" type="double" value="0.5"/>
        <param name="max_value" type="double" value="18.0"/>
        # Statistical Outlier Removal Filter parameters
        <param name="meanK" type="int" value="75"/>
        <param name="mulThresh" type="double" value="1.25"/>
    </node>
</launch>
